juriscraper/opinions/united_states/state/nc.py

```diff
@@ -199,3 +163,33 @@ def make_backscrape_iterable(self, kwargs: dict) -> None:
         end = int(end) + 1 if end else self.current_year
 
         self.back_scrape_iterable = range(start, end)
```

Comment:
if you add `self._process_html()` to the download backwards it helps negate the downloads method